### PR TITLE
[Home] New Orchestration section on homepage

### PR DESCRIPTION
**Pull request by ritiksaxena124**
**Description**
1. This PR adds a new section on the homepage for orchestration
2. Two-column layout used for diagramming and orchestration.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0b6263e830e00bf152adf5d38e4840db5b5c44f8 at: https://65903397b4d60c78822e621f--layer5.netlify.app

**Comment by leecalcote** [comment]
sweet

File: src/components/Features/TwoColLayout.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  return (
+    <Section>
+      <Container>
+        <ImageWrapper ref={containerRef}>
+          <Link to="/whiteboard">
+            <img src={isDark ? DiagrammingImageDark : DiagrammingImageLight} alt="image" />
+          </Link>
+        </ImageWrapper>
+        <ContentWrapper>
+          <h2>Simplify Your Visual Workflow</h2>
```
Comment (leecalcote) [inline src/components/Features/TwoColLayout.js:29]:
```suggestion
          <h2>Visualize and Simplify Platform Engineering</h2>
```

**Comment by l5io** [comment]
🚀 Preview for commit 6a26ed1c0ad20d1233eef7344b5eed26b38d3c7c at: https://659286fc8522cba2d6701351--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit cc07e6904b9ea4c7b80f808ed40a1db197e9e705 at: https://6592d7205e2c05d03cfe00cf--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7f1d91e0f10e9fa661d4b00e33bce0decd6d48a0 at: https://6593fc3aa4dda4008ab40108--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c4eb2982e296197a23be0883b1f53d1e23948d28 at: https://65940a64b4ecbc094e3d7792--layer5.netlify.app

**Pull request merged**